src/java.base/macosx/classes/sun/nio/ch/KQueuePort.java

```diff
@@ -50,7 +54,7 @@ final class KQueuePort
     private final int kqfd;
 
     // address of the poll array passed to kqueue_wait
-    private final long address;
+    private final MemorySegment pollArrayRegions;
```

Comment:
The "Regions" suffix is confusing here? This is the poll array that we invoke kqueue so I don't think "Regions" should be in the name.